test-files/typescript-errors.ts

```
@@ -0,0 +1,60 @@
+
+// Poor naming
+const a = true
+const b = false
+
+// Missing return type annotation
+function badFunction(param) {
+  return param + 1
+}
+
+// Using var instead of const/let
+var oldStyleVariable = 'bad'
```

ESLint

[error] 'oldStyleVariable' is assigned a value but never used. Allowed unused vars must match /^_$/u.

(@typescript-eslint/no-unused-vars)